yang/ietf-bearer-svc.yang

```diff
@@ -91,48 +160,67 @@ module ietf-bearer-svc {
         description
           "A description of this bearer.";
       }
-      container customer-device {
-        description
-          "Identification of the customer device that terminates the
-           bearer.";
-        leaf device-id {
-          type string;
-          description
-            "Identifier for the device.";
-        }
-        container location {
+
```

Comment:
```suggestion
  leaf op-comment {
    type string;
    description
      "Includes comments that can be shared with operational teams and
        which may be useful for the activation of a bearer. This may include, 
        for example, information about the building, level, etc.";
  }
```